ProjectLighthouse/Database.cs

```diff
@@ -312,6 +317,40 @@ public async Task UnqueueLevel(int userId, Slot queuedSlot)
         await this.SaveChangesAsync();
     }
 
+    public async Task BlockUser(int userId, User blockedUser)
+    {
+        if (userId == blockedUser.UserId)
+        {
+            return;
+        }
+        
+        User? user = await this.Users.FirstOrDefaultAsync(u => u.UserId == userId);
```

Comment:
```suggestion
        User? user = await this.Users.FindAsync(u => u.UserId == userId);
```
I've been meaning to change a lot of the FirstOrDefault calls that don't use .Include because FindAsync can apparently work better for finding entities by primary key and can reduce database trips if the entity is already tracked.